.github/workflows/pr_lint_pr_title.yml

```diff
@@ -43,6 +43,7 @@ jobs:
             l1
             l2
```

Comment:
and maybe you want to add a special label here: https://github.com/lambdaclass/ethrex/blob/main/.github/scripts/set-pr-labels.js#L22-L27